### ISSUE TITLE
Discuss NASA dataset

### ISSUE THREAD

**Issue by kveerama** [signals-dev/Orion#116]
Opening a thread to discuss ```NASA``` dataset. We can add comments and discuss its characteristics, suitability for using it for benchmarking.

**Comment by sajinpgupta** [comment]
Can this code be used for multi variate dataset?

**Comment by sarahmish** [comment]
@sajinpgupta yes! Checkout our multivariate tutorial https://github.com/sintel-dev/Orion/blob/master/tutorials/Multivariate_Input.ipynb

Also related #265.
